src/api/pool_api.js

```diff
@@ -121,6 +129,10 @@ module.exports = {
                             },
                         }
                     },
+                    // This boolean field dictates whether the RPC requester should be saved in the DB as the owner of the resource
```

Comment:
Why can't we store it for every resource and eliminate this flag?

Reply:
Upgrades?